newdocs/source/gettingstarted.rst

```diff
@@ -0,0 +1,159 @@
+Getting started
+===============
+.. attention:: These guidelines are written for developers to install *hazen* on MacOS or Linux and are a work in progress!
+
+If you would like to use with *hazen* via the CLI or contribute to its development, this is the installation guide for you.
+
+If, however, you are interested in using or contributing to the *hazen web app*, please visit that project's GitHub repository `here <https://github.com/GSTT-CSC/hazen-web-app>`_ for its installation guide.
+
+Prerequisites
+-------------
+
+* Python v3.9
+* Git
+* Docker
+
+Install
+-------
+
+Clone the repository
+^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+   git clone https://github.com/GSTT-CSC/hazen.git
+
+Docker
+^^^^^^^^^^^^^^
+We recommend using the Docker version of *hazen* as it is easy to get up-and-running and is linked to the most stable release. Refer to the `Docker installation instructions <https://docs.docker.com/engine/install>`_ to install Docker on your host computer.
```

Comment:
we can leave this for now, but in future it'd be nice to be able to say we recommend the web app version of hazen!

Reply:
What do you think of adding a tip at the top of the page? For example:

<img width="707" alt="image" src="https://user-images.githubusercontent.com/67117138/184177210-46d88dfd-2c69-4c11-b163-f4269d648b83.png">

@tomaroberts Is there a new Lightsail link we can use or should I hold off for now/use the Heroku link?

Reply:
We still need to decide on our long-term deployment strategy. I've shown POC that Lightsail will work. We need to work out a few details such as payment method and potential URL – Lightsail allows you to use a static IP, but would be preferable to point hazen.com or hazen.gstt.nhs.uk at it instead. Can obviously just use static IP in short-term if getting a URL is a challenge.